src/pages/Mypage/Mypage.js

```diff
@@ -1,7 +1,121 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import MatchTagWrapper from '../../components/MatchTagWrapper/MatchTagWrapper';
+import { FaGreaterThan } from 'react-icons/fa';
+import filterTextInfo from '../../utils';
+import styled from 'styled-components';
 
 function Mypage() {
-  return <div className="mypage" />;
+  const [userData, setUserData] = useState({});
+  useEffect(() => {
+    fetch('/data/userData.json')
+      .then(res => res.json())
+      .then(data => setUserData(data.result));
+  }, []);
+
+  return (
+    <div className="mypage">
+      <Container>
+        <MainTitle>계정 설정</MainTitle>
+        <ProfileImageWrapper>
+          <ProfileImage src="/images/mangorookie.jpeg" alt="profileImage" />
+        </ProfileImageWrapper>
+        <ProfileInfoWrapper>
+          <AccountInfo>
+            {filterTextInfo(userData).map(data => {
+              return (
+                <ItemContainer key={data.id}>
+                  <Item>
+                    <Title>{data[0].slice(5).toUpperCase()}</Title>
+                    <Info>{data[1]}</Info>
```

Comment:
```suggestion
     const [title, info] = data;
~~~
                    <Title>{title.slice(5).toUpperCase()}</Title>
                    <Info>{info}</Info>
```
data[0]. data[1] 이렇게 되어있으면 어떤 데이터인지 코드만 보고 알아보기 힘들기에 구조분해할당하면서 이름을 붙여주면 훨씬 가독성이 좋은 코드가 됩니다!